plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -195,6 +195,13 @@
     @Parameter(property = "fabric8.gitRemote")
     private String gitRemote;
 
+
+    /**
+     * The domain added to the service ID when creating OpenShift routes
+     */
+    @Parameter(property = "fabric8.domain")
```

Comment:
An entry in the documentation for this property would be nice

Reply:
https://github.com/fabric8io/fabric8-maven-plugin/pull/1730/files#diff-d71102b809959f854348a4454d17ecebL97

I basically shifted this property from ApplyMojo to ResourceMojo. The Usage for the user should not be affected.

Reply:
Ok.
Anyway, for JKube we should consider having documentation for every Mojo property.

Reply:
Yes, for sure!

Reply:
bdw, I don't see any pre-existing documentation related to this property.